colour.py

```diff
@@ -0,0 +1,41 @@
+class RGBColour:
```

Comment:
`__eq__` and `__hash__` could be added. Will help if we ever need to use these as dict keys.